certified-connectors/Vonage/apiDefinition.swagger.json

```diff
@@ -0,0 +1,804 @@
+            "x-ms-summary": "If you do not provide `number` in international format or you are not sure if `number` is correctly formatted, specify the two-character country code in `country`. Verify will then format the number for you",
+            "in": "formData",
+            "name": "country",
+            "type": "string"
+          },
+          {
+            "description": "An 18-character alphanumeric string you can use to personalize the verification request SMS body, to help users identify your company or application name. For example: \"Your `Acme Inc` PIN is ...\"",
+            "x-ms-summary": "An 18-character alphanumeric string you can use to personalize the verification request SMS body, to help users identify your company or application name",
+            "in": "formData",
+            "maxLength": 18,
+            "name": "brand",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "default": "VERIFY",
+            "description": "An 11-character alphanumeric string that represents the [identity of the sender](https://developer.nexmo.com/messaging/sms/guides/custom-sender-id) of the verification request. Depending on the destination of the phone number you are sending the verification SMS to, restrictions might apply",
+            "x-ms-summary": "An 11-character alphanumeric string that represents the [identity of the sender](https://developer.nexmo.com/messaging/sms/guides/custom-sender-id) of the verification request. Depending on the destination of the phone number you are sending the verification SMS to, restrictions might apply",
+            "in": "formData",
+            "maxLength": 11,
+            "name": "sender_id",
+            "type": "string"
+          },
+          {
+            "default": 4,
+            "description": "The length of the verification code",
+            "x-ms-summary": "The length of the verification code",
+            "enum": [
+              4,
+              6
+            ],
+            "in": "formData",
+            "name": "code_length",
+            "type": "integer"
+          },
+          {
+            "default": "en-us",
+            "description": "By default, the SMS or text-to-speech (TTS) message is generated in the locale that matches the `number`. For example, the text message or TTS message for a `33*` number is sent in French. Use this parameter to explicitly control the language used for the Verify request. A list of languages is available: <https://developer.vonage.com/verify/guides/verify-languages>",
+            "x-ms-summary": "By default, the SMS or text-to-speech (TTS) message is generated in the locale that matches the `number`. For example, the text message or TTS message for a `33*` number is sent in French. Use this parameter to explicitly control the language used for the Verify request. A list of languages is available: <https://developer.vonage.com/verify/guides/verify-languages>",
```

Comment:
Please check this summary. These summaries should not exceed 80 characters limit. That's why the swagger validation fails.

Reply:
Addressed all summaries that were over 80 chars